### ISSUE TITLE
New Check - count of stonith devices

### ISSUE THREAD

**Issue by Darryl-G** [Azure/SAP-on-Azure-Scripts-and-Utilities#61]
Hello.

I'd like to ask for an additional check to be added into the qualitycheck v2 .

On SUSE pacemaker clusters, some of the old MS docs may have given the impression that a stonith device per cluster node is needed.
There should only be 1 stonith device in the cluster when using fence_azure_arm.

OS: SUSE
OSVersion: All
HighAvailability: true
CommandType: OS
ProcessingCommand:  `crm config show | grep 'stonith:fence_azure_arm' | wc -l`
ExpectedResult: 1
RootRequired: true
AdditionalInfo: Checking stonith device count is 1.

As a side note, some of the checks that look at the corosync config could get the info using the crm command like so:
`crm corosync get totem.token`

Darryl

**Comment by msftphleiten** [comment]
this scenario depends on the implemented operating system, there is a new option to run fence_kdump on Red Hat, this will be implemented as an optional check in the future

**Comment by Darryl-G** [comment]
Thanks.  The issue I've raised is specific to fence_azure_arm.
I have inherited approx 10 pacemaker clusters, all have more than one fence_azure_arm device, each prefixed with the name of the nodes in the clusters.
I also reported this issue to Bartosz because one of his blogs also had the same problem.  I can only presume that an old version of the MS docs provided steps for setting up 1 fence_azure_arm device for each cluster node.  At some point the MS docs changed and now only show 1 fence_azure_arm in each cluster.
As part of the QualityCheck I think it would be useful to know if the number of fence_azure_arm is > 1 and therefore an incorrect setup.


Darryl
